lighthouse-core/test/lib/traces/tracing-processor.js

```diff
@@ -39,6 +39,12 @@ function createRiskPercentiles(percentiles, times) {
 }
 
 describe('TracingProcessor lib', () => {
+  it('doesn\'t throw when module is loaded', () => {
+    assert.doesNotThrow(_ => {
```

Comment:
is this test necessary? If it throws when it's loaded at the top level that would fail the unit tests as well.